provd/internal/services/hello/hello.go

```diff
@@ -0,0 +1,32 @@
+package hello
```

Comment:
each package too should have one comment, but I’m fine skipping this for now and you will then validate golangci-lint with it :)